tutoraspects/patches/openedx-common-settings

```diff
@@ -4,7 +4,7 @@ EVENT_SINK_CLICKHOUSE_BACKEND_CONFIG = {
 	"username": "{{ ASPECTS_CLICKHOUSE_CMS_USER }}",
 	"password": "{{ ASPECTS_CLICKHOUSE_CMS_PASSWORD }}",
 	"database": "{{ ASPECTS_EVENT_SINK_DATABASE }}",
-	"timeout_secs": {{ ASPECTS_EVENT_SINK_CLICKHOUSE_TIMEOUT_SECS }}
```

Comment:
I'd like to keep this one

Reply:
this one can be overridden with `openedx-common-settings`

Reply:
But doing so requires either overwriting the entire config manually or replacing the key, which is weird since we allow all of the other config for this setting to happen in one place. I think it makes the most sense to keep everything for one setting in the same place.